embulk-output-redshift/src/main/java/org/embulk/output/redshift/RedshiftCopyBatchInsert.java

```diff
@@ -94,6 +97,20 @@ public RedshiftCopyBatchInsert(JdbcOutputConnector connector,
                     + " IAM user needs \"s3:GetBucketLocation\" permission if Redshift region and S3 region are different.");
         }
         this.s3RegionName = s3RegionName;
+
+        String copyUnloadIamRoleARN = null;
+        if (copyUnloadIamRoleName != null && copyUnloadIamRoleName.length() > 0) {
+            String accountId = null;
+            if (copyUnloadIamAccountId != null && copyUnloadIamAccountId.length() > 0) {
+                accountId = copyUnloadIamAccountId;
+            } else {
+                GetCallerIdentityRequest request = new GetCallerIdentityRequest();
+                GetCallerIdentityResult response = this.sts.getCallerIdentity(request);
+                accountId = response.getAccount();
+            }
+            copyUnloadIamRoleARN = "arn:aws:iam::"+accountId+":role/"+copyUnloadIamRoleName;
```

Comment:
The local variable 'copyUnloadIamRoleARN' is unnecessary. You can simply set 'this.copyUnloadIamRoleARN' in this line.

Reply:
Thank you to review!
Exactly it is no need local variable. I fix it.